bjoern/_bjoernmodule.c

```diff
@@ -53,7 +70,22 @@ PyMODINIT_FUNC init_bjoern(void)
   assert(FileWrapper_Type.tp_flags & Py_TPFLAGS_READY);
   PyType_Ready(&StartResponse_Type);
   assert(StartResponse_Type.tp_flags & Py_TPFLAGS_READY);
+  Py_INCREF(&FileWrapper_Type);
+  Py_INCREF(&StartResponse_Type);
 
+#if PY_MAJOR_VERSION >= 3
+  PyObject* bjoern_module = PyModule_Create(&module);
+  if (bjoern_module == NULL) {
+	  return NULL;
+  }
+
+  PyModule_AddObject(bjoern_module, "version", Py_BuildValue("(iii)", 1, 4, 3));
+  PyModule_AddObject(bjoern_module, "FileWrapper", (PyObject *)&FileWrapper_Type);
```

Comment:
I will remove the FileWrapper. I think we should expose the version tuple. But instead of exposing it in _bjoern, it probably should better promoted to the top level bjoern module.